lib/jara/cli.rb

```diff
@@ -0,0 +1,57 @@
+# encoding: utf-8
+
+require 'jara'
+require 'optparse'
+
+module Jara
+  class Cli
+    def initialize(argv)
+      @command = argv.first
+      @argv = argv.drop(1)
+    end
+
+    def run
+      parse_argv(@argv)
+      options = {}
+      options[:archiver] = @archiver if @archiver
+      options[:build_command] = @build_command if @build_command
+      releaser = Jara::Releaser.new(@environment, @bucket, options)
+      case @command
+      when /help|-h/
```

Comment:
It's supported: `/help/` matches `--help`, the extra `/-h/` is there to support only "-h" (it matches `--hello` also, but I figured that doesn't really matter).